packages/client/src/index.ts

```diff
@@ -0,0 +1,42 @@
+import { Command } from "commander";
```

Comment:
What is this file doing? Maybe it will be great if we explain about the following code in the beginning of the document.

Reply:
done